src/lang/en.json

```diff
@@ -0,0 +1,59 @@
+{
+  "breadcrumbs": {
+    "accountDashboard": "BC Registries Dashboard",
+    "registryDashboard": "My Business Registry"
+  },
+  "labels": {
+    "general": {
+      "email": "Email",
+      "phone": "Phone"
+    },
+    "business": {
+      "businessNum": "Business Number",
+      "incorporationNum": "Incorporation Number",
+      "registrationNum": "Registration Number",
+      "legalTypes": {
+        "BEN": "BC Benefit Company",
+        "BC": "BC Limited Company",
+        "CP": "BC Cooperative Association",
+        "SP": "Sole Proprietorship",
+        "GP": "BC General Partnership",
+        "undefined": "N/A"
+      }
+    },
+    "services": {
+      "bcsc": "BC Services Card",
+      "bceid": "BCeID",
+      "idir": "IDIR"
+    }
+  },
+  "header": {
+    "title": "BC Registry and Online Services",
+    "buttons": {
+      "createAccount": "Create account"
+    },
+    "menus": {
+      "headers": {
+        "accountSettings": "Account Settings",
+        "selectLogin": "Select log in method",
+        "switchAccount": "Switch Account"
+      },
+      "labels": {
+        "accountInfo": "Account Info",
+        "createAccount": "Create Account",
+        "editProfile": "Edit Profile",
+        "logOut": "Log Out",
+        "teamMembers": "Team Members",
+        "transactions": "Transactions"
+      }
+    }
+  },
+  "footer": {
+    "labels": {
```

Comment:
I think we are mixing again.
we have 
`labels -> general -> email`
and we have
`footer -> labels -> ....`

Reply:
right -- was thinking we would shift some things logically for the header/footer so that its contained a bit differently and doesn't ever conflict with the project at hand (for when we import these from layers). This would mean some things would have their own section so they are always separate and never conflict when the en.json is merged. That was my thinking here

Reply:
updated en.json as discussed